dockstore-webservice/src/main/java/io/dockstore/webservice/resources/UserResource.java

```diff
@@ -273,6 +275,8 @@ private void deleteSelfFromEntries(User user) {
                                 MessageFormat.format("Unexpected entry type {0}", entry.getClass().toString()),
                                 HttpStatus.SC_INTERNAL_SERVER_ERROR);
                     }
+                    List<Event> eventsByEntryIDs = eventDAO.findEventsByEntryId(entry.getId());
```

Comment:
EventDAO can have a delete by user for example, done via JPA delete.

Reply:
Originally, the ones where results are user-facing have limit and offset. The ones that aren't user-facing do not for fear of doing a partial cleanup job. Currently, the implementation already has a delete function. It provides more functionality to have a simple find function and then use the delete function. 

The alternative to only have the delete function in the DAO won't really differ much because that'll not have limit and offset either.